Content.Server/_White/EmpFlashlight/EmpOnHitComponent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+using Content.Server._White.EmpFlashlight;
+using Content.Server.Emp;
+
+namespace Content.Server._White.EmpFlashlight;
+
+/// <summary>
+/// Upon being triggered will EMP target.
+/// </summary>
+[RegisterComponent]
+[Access(typeof(EmpHitSystem))]
+
+public sealed partial class EmpOnHitComponent: Component
+{
+    [DataField("range"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

Если строковое название равно названию переменной- его можно не писать

Reply:
Оставил, т.к. это полная копия старого компонента - EmpOnTriggerComponent.

Лучше не добавлять путаницы.